Put back a couple settings for Kiali that were accidentally deleted.

johnma14 opened the pull request:
Some Kiali settings were accidently deleted when the new installation options for
release-1.1 was published. This is because these settings were commented out in
the values.yaml file for kiali under istio/kubernetes/helm/istio/charts/kiali.

Bug:#3660

istio-testing commented:
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/istio/istio/pull/12472#" title="Author self-approved">johnma14</a>*
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **sdake**

If they are not already assigned, you can assign the PR to them by writing `/assign @sdake` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[install/OWNERS](https://github.com/istio/istio/blob/release-1.1/install/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":["sdake"]} -->

jmazzitelli commented:
/lgtm

istio-testing commented:
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/istio/istio/pull/12472#issuecomment-472596276" title="LGTM">jmazzitelli</a>*, *<a href="https://github.com/istio/istio/pull/12472#" title="Author self-approved">johnma14</a>*
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **sdake**

If they are not already assigned, you can assign the PR to them by writing `/assign @sdake` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[install/OWNERS](https://github.com/istio/istio/blob/release-1.1/install/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":["sdake"]} -->

johnma14 commented:
This is related to an issue that was opened in istio.io: https://github.com/istio/istio.io/pull/3669.

istio-testing commented:
@johnma14: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
prow/istio-integ-k8s-tests.sh | 1e63cc5d0f878884fefb4ed798f147c4d45590ca | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/12472/istio-integ-k8s-tests/4940) | `/test istio-integ-k8s-tests`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@johnma14: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
prow/istio-integ-k8s-tests.sh | 1e63cc5d0f878884fefb4ed798f147c4d45590ca | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/12472/istio-integ-k8s-tests/4940) | `/test istio-integ-k8s-tests`
prow/istio-pilot-multicluster-e2e.sh | 1e63cc5d0f878884fefb4ed798f147c4d45590ca | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/12472/istio-pilot-multicluster-e2e/9497) | `/test istio-pilot-multicluster-e2e`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@johnma14: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
prow/istio-pilot-multicluster-e2e.sh | 1e63cc5d0f878884fefb4ed798f147c4d45590ca | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/12472/istio-pilot-multicluster-e2e/9497) | `/test istio-pilot-multicluster-e2e`
prow/istio-integ-k8s-tests.sh | 97998b443d96c905bf79d023cea7bab8541ddf41 | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/12472/istio-integ-k8s-tests/5021) | `/test istio-integ-k8s-tests`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

istio-testing commented:
@johnma14: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
prow/istio-integ-k8s-tests.sh | 97998b443d96c905bf79d023cea7bab8541ddf41 | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/12472/istio-integ-k8s-tests/5021) | `/test istio-integ-k8s-tests`
prow/istio-pilot-multicluster-e2e.sh | 97998b443d96c905bf79d023cea7bab8541ddf41 | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/12472/istio-pilot-multicluster-e2e/9563) | `/test istio-pilot-multicluster-e2e`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

johnma14 commented:
/retest

johnma14 commented:
Not sure if the tests are failing because of my changes. There were some tabs in the file that I modified which was causing the linter to complain but I fixed those issues and the linter tests are running fine.

johnma14 commented:
/retest

istio-testing commented:
@johnma14: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
prow/istio-pilot-multicluster-e2e.sh | 97998b443d96c905bf79d023cea7bab8541ddf41 | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/12472/istio-pilot-multicluster-e2e/9563) | `/test istio-pilot-multicluster-e2e`
prow/istio-integ-k8s-tests.sh | 97998b443d96c905bf79d023cea7bab8541ddf41 | [link](https://k8s-gubernator.appspot.com/build/istio-prow/pr-logs/pull/istio_istio/12472/istio-integ-k8s-tests/5102) | `/test istio-integ-k8s-tests`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

johnma14 commented:
/retest

istio-testing commented:
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/istio/istio/pull/12472#issuecomment-472596276" title="LGTM">jmazzitelli</a>*, *<a href="https://github.com/istio/istio/pull/12472#" title="Author self-approved">johnma14</a>*, *<a href="https://github.com/istio/istio/pull/12472#pullrequestreview-218373264" title="Approved">linsun</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[install/OWNERS](https://github.com/istio/istio/blob/release-1.1/install/OWNERS)~~ [linsun]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":[]} -->
